services/filemanager/filemanager.go

```diff
@@ -44,6 +44,7 @@ type FileManager interface {
 	ListFilesWithPrefix(ctx context.Context, prefix string, maxItems int64) (fileObjects []*FileObject, err error)
 	GetConfiguredPrefix() string
 	SetTimeout(timeout *time.Duration)
+	Dispose()
```

Comment:
`Close` is a more common method name for performing necessary cleanup operations before disposing an object

Reply:
done